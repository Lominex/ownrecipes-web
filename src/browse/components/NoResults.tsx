import { Col, Row } from 'react-bootstrap';
import { defineMessages, useIntl } from 'react-intl';

const NoResults: React.FC = () => {
  const intl = useIntl();
  const messages = defineMessages({
    no_results: {
      id: 'browse.no_results',
      description: 'No results header',
      defaultMessage: 'There are no results for your search.',
    },
  });

  return (
    <Row>
      <Col xs={12}>
        <Row id='browse'>
          <div className='spinner'>
            <h3 className='no-results'>{intl.formatMessage(messages.no_results)}</h3>
          </div>
        </Row>
      </Col>
    </Row>
  );
};

export default NoResults;
