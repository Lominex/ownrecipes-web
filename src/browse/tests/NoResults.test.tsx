import NoResults from '../components/NoResults';
import create from '../../test/create';

test('NoResults component test', () => {
  const component = create(
    <NoResults />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
