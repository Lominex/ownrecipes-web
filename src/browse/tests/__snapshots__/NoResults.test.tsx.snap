// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResults component test 1`] = `
<p
  className="body1 no-results placeholder"
>
  There are no results for your search.
</p>
`;
